src/NServiceBus.AcceptanceTests/MessageId/When_message_has_no_id_header.cs

```diff
@@ -5,7 +5,7 @@
     using System.Threading.Tasks;
     using AcceptanceTesting;
     using EndpointTemplates;
-    using Pipeline;
+    using NServiceBus.Pipeline;
```

Comment:
but still seems like it needs to have the fully qualified namespace otherwise I get a build error.